src/gateways/createHighlightClient.spec.ts

```diff
@@ -1,7 +1,12 @@
 import { GetHighlightsSourceTypeEnum } from '@openstax/highlighter/dist/api';
+import { UnauthenticatedError } from '../app/types';
 import { resetModules } from '../test/utils';
 import createHighlightClient from './createHighlightClient';
 
+jest.mock('../app/utils', () => ({
+  ...jest.requireActual('../app/utils'),
+}));
+
```

Comment:
Is this necessary? Won't `jest.mock('../app/utils')` work?

Reply:
actually this is mocking it with the actual module, so wouldn't just not mocking it do the same?